Add CR90 and list it under CM35

kristovatlas opened the pull request:
Added CR90
(71b9d459fb307c0a24e9fffad8486e7436341fd9e0bf08c25d47e0e4ee8f69f7) and
listed it under CM35
(8f1ac060930a5fe4286ce57d1096ef51b63a65a6dd36926a0cbb89c427473aeb) for
RBF stuff.

This fulfills
https://github.com/OpenBitcoinPrivacyProject/wallet-ratings/issues/36
for now.

kristovatlas committed:
Add CR90 and list it under CM35

Added CR90
(71b9d459fb307c0a24e9fffad8486e7436341fd9e0bf08c25d47e0e4ee8f69f7) and
listed it under CM35
(8f1ac060930a5fe4286ce57d1096ef51b63a65a6dd36926a0cbb89c427473aeb) for
RBF stuff.

This fulfills
https://github.com/OpenBitcoinPrivacyProject/wallet-ratings/issues/36
for now.

kristovatlas commented:
meeting notes: let's split up the criteria into 2

dcousens commented:
ACK if split into 2

kristovatlas committed:
Merge branch 'reorg-to-scoring-system' into issue-36-RBF-criterion

resolve order conflict of criteria in obpp3.json

kristovatlas committed:
split CR90 into two criteria

split CR90
(71b9d459fb307c0a24e9fffad8486e7436341fd9e0bf08c25d47e0e4ee8f69f7) into
two criteria; second one is CR91
c9c2f71a47d5aa66336c8c126aba9b44688d6b06e7e65f53023e6334d5c63fb7).

dcousens commented on obpp3.json:
```diff
@@ -2376,9 +2380,14 @@
-         "id": "OBPPV3/CR90",
-            "description": "More than one version of a transaction is never signed, or new versions do not modify transaction output values from previous versions",
+            "id": "OBPPV3/CR90",
+            "description": "The wallet client never signs and broadcasts more than one version of a transaction (RBF)",
```
`s/RBF/No RBF support/`?

dcousens commented on obpp3.json:
```diff
@@ -2376,9 +2380,14 @@
+        },
+        {
+            "id": "OBPPV3/CR91",
+            "description": "When creating new versions of RBF transactions, the client does not modify transaction output values from previous versions",
```
ACK

kristovatlas committed:
correct validation-breaking mismatch in CR90 description

fixed description of CR90
(71b9d459fb307c0a24e9fffad8486e7436341fd9e0bf08c25d47e0e4ee8f69f7).

@dcousens I will be tagging CR90 and CR91 with “RBF” once I merge the
tags branch — I think this will look nicer than adding parenthetical
comments :-)

dcousens commented:
ACK

kristovatlas commented:
Merge resolved. I'm merging this since was previously ACK'd.

Commit:
merge and resolving re-ordering conflict in obpp3.json
